image-search/ViewControllers/ViewController.swift

```diff
@@ -40,14 +46,24 @@ class ViewController: UIViewController {
     
         @IBAction func alterText(_ sender: Any) {
```

Comment:
Identation. All this function is a "tab" forwarded.

Reply:
Changed